.github/workflows/acceptance-tests.yml

```diff
@@ -185,6 +188,30 @@ jobs:
           TEST_REGEX: "^TestAccClusterRSCluster"
         run: make testacc
 
+  search_deployment:
+    needs: [ change-detection ]
+    if: ${{ needs.change-detection.outputs.search_deployment == 'true' || github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' || github.event.label.name == 'run-testacc' || github.event.label.name == 'run-testacc-search-deployment' || inputs.parent-event-name == 'release' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: 'go.mod'
+      - name: Acceptance Tests
+        env:
+          MONGODB_ATLAS_PUBLIC_KEY: ${{ secrets.MONGODB_ATLAS_PUBLIC_KEY_CLOUD_DEV }}
```

Comment:
CI: true is missing ;-).   cc @marcosuma

Reply:
true, however that env variable is not used in this job as no tests need to be skipped (several jobs dont define it). Can add just for consistency.

Reply:
yeah, i know it's only used when calling SkipTestForCI , it was more to advocate for a common env zone :-)

Reply:
will try to answer to your PR :) but I think the global env variables would make the opposite effect here when you create a new job and you don't immediately expect there are global variables that might affect this job's behaviour

Reply:
or it could be the opposite, that you try to use SkipTestForCI and don't know why it's not skipping it.

I think it depends on the expectations set on the tests run from that GH action. for me it's clear in this case that all tests "want" to have CI:true even if they don't use/need it right now.